RealmSwift/Util.swift

```diff
@@ -47,6 +47,110 @@ internal func gsub(pattern: String, template: String, string: String, error: NSE
                                            withTemplate: template)
 }
 
+// MARK: ObjectiveCBridgeable
+
+// Used for conversion from Objective-C types to Swift types
+internal protocol ObjectiveCBridgeable  {
+    /* FIXME: Remove protocol once SR-2393 bridges all integer types to `NSNumber`
```

Comment:
A wild [proposal](https://github.com/jckarter/swift-evolution/blob/be49e08f56450ffea394306198bcd25f58915e30/proposals/XXXX-bridge-nsnumber-and-nsvalue.md) appeared—looks likely we'll get a fix! 😎